### PR TITLE
CRYP-26: refactor to single get transactions call and fix dates

### PR DESCRIPTION
**Pull request by jason-gerard**
JIRA: https://cryptify.atlassian.net/browse/CRYP-26

### CODE REVIEW

File: packages/client/components/TransactionList.tsx [hunk 1]
```diff
@@ -49,10 +49,9 @@ export function TransactionList({ transactions, walletAddress, displaySeparation
         return (
             <>
                 {transactions.map((transaction, i) => (
-                    <>
+                    <Box key={i}>
```
Comment (alexbara2000) [inline packages/client/components/TransactionList.tsx:52]: What does this change?

File: packages/client/components/TransactionListItem.tsx [hunk 2]
```diff
@@ -18,36 +19,28 @@ type Props = {
...
-        return `${address.substring(0, 6)}...${address.substring(address.length - 4, address.length)}`;
-    }
-
     function getCurrencyType(): string {
         return transaction.transactionAddress.substring(0, 2) == "0x" ? "ETH" : "BTC";
     }
 
-    function formatAmount(amount: string): string {
-        return amount.length > 15 ? amount.substring(0, 15) + "..." : amount;
+    function getFormattedDate(timestamp: string): string {
```
Comment (alexbara2000) [inline packages/client/components/TransactionListItem.tsx:26]: Is this only used in this file?

**Review by alexbara2000 — APPROVED** [review]
Looks good, I am relatively impressed!

File: packages/client/components/TransactionList.tsx (hunk 1 above)
Comment (jason-gerard) [reply packages/client/components/TransactionList.tsx:52]: Fixes the error of each mapped item not having a unique key, you had put it on the wrong element

File: packages/client/components/TransactionListItem.tsx (hunk 2 above)
Comment (jason-gerard) [reply packages/client/components/TransactionListItem.tsx:26]: As of now, yes

**Pull request merged**